frontend/src/app/person/person.component.html

```diff
@@ -72,12 +101,21 @@ <h2>Contact</h2>
       <div>{{ person.phoneNumber }}</div>
     </div>
 
-    <ng-container *ngIf="person.mediationEnabled">
-      <h2>Logement</h2>
+    <div class="d-flex mb-2" id="accompanying"  *ngIf="person.mediationEnabled">
+      <div class="mr-3"><span class="fa fa-users" title="accompagnant"></span></div>
```

Comment:
Not fan of this icon. I always see an alsacian woman instead of a group of users when I see it :-). Maybe http://fontawesome.io/icon/universal-access/

Reply:
;-)

Reply:
:+1: